src/angular-app/languageforge/lexicon/editor/field/dc-audio.component.ts

```diff
@@ -144,7 +144,8 @@ export class FieldAudioController implements angular.IController {
 
   audioRecorderCallback = (blob: Blob) => {
     if (blob) {
-      const fileName = 'recording_' + moment.utc().format('YYYY_MM_DD_HH_mm_ss') + '.mp3';
+      const date = new Date();
+      const fileName = 'recording_' + format(addMinutes(date, date.getTimezoneOffset()), 'YYYY_MM_DD_HH_mm_ss') + '.mp3';
```

Comment:
The logic is correct here, but `date-fns` uses the date formatting symbols from the [Unicode standard](https://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table), where `YYYY` and `DD` don't mean what you probably expect them to mean. See https://github.com/date-fns/date-fns/blob/master/docs/unicodeTokens.md for a broader explanation. The correct format string to use here is `'yyyy_MM_dd_HH_mm_ss'`. With that format string, this will produce the same filenames that it used to with the moment.js function.